scripts/spell_check/spelling.dat

```diff
@@ -2390,6 +2390,8 @@ dosent':doesn't
 dosen't:doesn't
 doub:doubt
 doulbe:double
+double click:double-click
+doubleclick:double-click
```

Comment:
@3nids Running check_spelling without adding the star doesn't catch more occurrences (no in-word proposed to fix - I suppose it means that they are not problematic).
Running check_spelling with `doubleclick:double-click:*` only returns an occurence of australia in the ... spelling.dat file.
I don't know what is going wrong...